Client/Extensions/UIWindow+Extension.swift

```diff
@@ -8,7 +8,7 @@ import Foundation
 extension UIWindow {
     static var keyWindow: UIWindow? {
         return UIApplication.shared.connectedScenes
-            .filter { $0.activationState == .foregroundActive }
+            .filter { $0.activationState == .foregroundActive || $0.activationState == .foregroundInactive }
```

Comment:
yes, I see that `keyWindow` is used in several key places of the app wondering if this change could cause side effects?

Reply:
Sure, I'll test that. For context: I noticed that `keyWindow` returned nil when I started multitasking. That is happening because the app is not in foreground while we select the second app to open. Because of that the whole sidebar/modal view gets messed up and displaying the wrong UI.

Reply:
Changed the code to use `attachedKeyWindow` instead of making this change. Thanks for your feedback 🙏